app/assets/javascripts/TortoiseJS/agent/component/input.coffee

```diff
@@ -127,8 +127,13 @@ window.RactiveInput = RactiveWidget.extend({
     """
     {{>input}}
     {{>contextMenu}}
+    {{# widget.boxedValue.type === 'Color' || widget.boxedValue.type === 'Number' }}
     <editForm idBasis="{{id}}" boxtype="{{widget.boxedValue.type}}"
+              display="{{widget.display}}" />
+    {{ else }}
+        <editForm idBasis="{{id}}" boxtype="{{widget.boxedValue.type}}"
               display="{{widget.display}}" isMultiline="{{widget.boxedValue.multiline}}" />
+    {{/}}
```

Comment:
It occurred to me last night (and I meant to tell you then) that this solution is a bit non-optimal.  We can avoid the code duplication here by doing something like what we do [here](https://github.com/NetLogo/Galapagos/blob/master/app/assets/javascripts/TortoiseJS/agent/component/editor.coffee#L23), where we just surround the optional property in an `if`.